action/protocol/staking/read_state.go

```diff
@@ -243,7 +243,7 @@ func getActiveBucketsCount(ctx context.Context, csr CandidateStateReader) (uint6
 		// after Greenland, read state from db
 		var total totalAmount
 		_, err := csr.SR().State(&total, protocol.NamespaceOption(StakingNameSpace), protocol.KeyOption(bucketPoolAddrKey))
-		if err != nil {
+		if err != nil && errors.Cause(err) != state.ErrStateNotExist {
```

Comment:
why need this check?
if pass Greenland and state does not exist, it is a real error that should not be escaped